### DISCUSSION TITLE
Weird background for syntax

### DISCUSSION THREAD

**Discussion by kentron** [atom/atom#23481] [Q&A]
Hi everyone, 
I'm trying to understand what generates this problem that seems to occur only with MacOS Monterrey. The background color for the code lines inside the main section get highlighted white and its impossible to read.  

Running same version of Atom and packages on other operating systems (and other versions of MacOS) show now problem.

I don't understand in which part of the documentation I could get guidelines to investigate why this happens.

Can someone help?

I paste part of the screenshot which shows the issue.

Thank you in advance,
![Screenshot Atom 1 58 0 weid background](https://user-images.githubusercontent.com/881275/149447148-7365d367-cafd-4ff8-a04a-c7ba82e1720c.png)
K

**Comment by icecream17** [comment]
what theme are you using?
